DiamondLightSource/SynchWeb
# LIMS-1059: Add visit column to shipments list

**JIRA ticket**: [LIMS-1059](https://jira.diamond.ac.uk/browse/LIMS-1059)

**Summary**:

Add a column to the /shipments page, showing the visits associated with a shipment (via the dewars).

**Changes**:
- Remove repeat select of 'prop'
- Add 'visit' to select, use group_concat(distinct ...)) to show multiple visits if needed
- Add 'visit' column to shipments page

**To test**:
- Load the shipments page of a test proposal, check 'Visit' column appears
- Check a shipment with no assigned visits has a blank field
- Check a shipment with 1 dewar and 1 visit has that visit displayed
- Check a shipment with multiple dewars, all on one visit, has that visit displayed (once)
- Check a shipment with multiple dewars and multiple visits has all visits displayed
